### PR TITLE
Monthly payment worksheet: fixes

### PR DESCRIPTION
**Pull request by virginiacc**
Monthly payment worksheet tweaks

## Additions

- Adds tabindex to worksheet section headers so summary section is part of tab order
- Adds "Know the Process" links to the related links section of worksheet

## Removals

- Removes `addEventListener` polyfill that prevents React from working in IE8

## Changes

- Makes inputs 100% width on tablet & mobile so they line up with right-aligned summary values (there's a max-width of 300px on mobile)
- Updates `related-links` template to allow for multiple urls per description, and updates all instances of `related-links` to use new format

## Testing

- Use Saucelabs to check that monthly payment worksheet works in IE8 now that the `addEventListener` polyfill has been removed

## Review

- @cfarm 
- @huetingj 

### Full-width inputs on tablet and mobile (SEE comments thread for updated images):

screen shot 2015-10-27 at 8 35 54 am

screen shot 2015-10-27 at 8 35 36 am

### Related links section on worksheet

screen shot 2015-10-27 at 1 54 02 pm

## Todos

- double check that other pages still work in IE8 without the `addEventListener` polyfill
- `addEventListener` polyfill was needed for the prepare worksheets, so those will need to be revised

## Checklist

* [X] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [X] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [X] Visually tested in supported browsers and devices 
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by huetingj** [comment]
Following our discussion during scrum: On mobile, let's set a max width of 50-ish percent? May need to be more like 60 percent? Would you mind also trying that for tablet— setting the max width for each column to around 75 percent?

**Comment by huetingj** [comment]
Also: it looks and works slick in general! :tada: wooo!

**Comment by virginiacc** [comment]
@huetingj This is about what 75% max-width would look like on iphone portrait:

screen shot 2015-10-27 at 9 45 30 am

**Comment by virginiacc** [comment]
@huetingj Here's what an input max-width of around 300px would look like at different screen widths:

### 320px:
screen shot 2015-10-27 at 1 00 54 pm

### 480px:
screen shot 2015-10-27 at 1 01 26 pm

### 599px:
screen shot 2015-10-27 at 1 01 41 pm

**Comment by huetingj** [comment]
This looks great, woohoo!! :+1:

File: src/closing-disclosure/index.html [hunk 1]
```diff
@@ -41,22 +41,36 @@ <h1 class="page-title">Closing Disclosure Explainer</h1>
         {% set process_links = [
             { 
               'desc': 'Already chosen a loan?',
-              'label': 'Get ready to close', 
-              'url': '/owning-a-home/process/close/'
+              'urls': [
```
Comment (cfarm) [inline src/closing-disclosure/index.html]: Can you add an issue to the backlog on github about these Related Links? Right now you have to edit every page that has them and this seems like something that should be fixed.

**Comment by cfarm** [comment]
Code reviewed and tested in IE8!

**Pull request merged**